spec/xpm_ruby/client_spec.rb

```diff
@@ -38,6 +38,21 @@ module XpmRuby
           expect(updated_client["Name"]).to eql(client["Name"])
         end
       end
+
+      describe ".list" do
+        let(:xero_tenant_id) { "0791dc22-8611-4c1c-8df7-1c5453d0795b" }
+        let(:access_token) { "token" }
+
+        it "list client" do
+          VCR.use_cassette("xpm_ruby/client/list") do
+            clients = Client.list(
```

Comment:
Maybe we can also test `detailed` and `modified_since` as well? At least check that we pass them through via a mock if they are presented?